src/window/mod.rs

```diff
@@ -45,6 +45,12 @@ impl Window {
         }
     }
 
+    /// Get raw SDL context for external use (e.g. audio context).
+    #[cfg(feature = "with-sdl")]
+    pub fn context(&mut self) -> sdl2::Sdl {
+        self.sdl_window.as_mut().unwrap().context()
```

Comment:
It's not great that this method can't be called before the first call to `update`, because `sdl_window` hasn't been created yet. We could refactor the code to create the SDL context in `Window::new` instead of `SdlWindow::new` to make this a bit easier to use.

Reply:
Okay. I think I got it.